azure-functions-sdk/src/commands/new_app.rs

```diff
@@ -117,6 +117,7 @@ impl<'a> NewApp<'a> {
                 json!({ "crate_version": env!("CARGO_PKG_VERSION") }),
             ),
             ("appsettings.json", "appsettings.json", json!({})),
+            ("local.settings.json", "local.settings.json", json!([])),
```

Comment:
Nit: I think generally the templates expect an object rather than an array (but the template doesn't use any values, so it's moot in this case, but it might be best to change this for consistency).